centos8/http/centos8.ks

```diff
@@ -47,6 +49,7 @@ python3-oauthlib
 rsync
 tar
 grub2-efi-x64
+grub2-efi-x64-modules
```

Comment:
grub2-efi-x64-modules is intentionally left out. CentOS 8 signs GRUB which comes from grub2-efi-x64. If grub2-efi-x64-modules is installed GRUB will be generated and unsigned. This breaks UEFI secure boot as GRUB must be signed with CentOS's key which we don't have access to.

Reply:
We've had several customers showing this particular issue as shown below in the curtin logs. After reproducing the issue myself, this was the best way to resolve it.

```
Stderr: grub2-install: error: /usr/lib/grub/x86_64-efi/modinfo.sh doesn't exist. Please specify --target or --directory.
```

Reply:
What version of Curtin are you using? Can you upload the log?

Reply:
curtin version 20.1-2-g42a9667f-0ubuntu1~18.04.1

Here's the link to the log from the maas server, which should have all the details. The errors at at the bottom of the page

https://paste.ubuntu.com/p/r9kjC5Fq2H/

Reply:
I think the boot failures may be due to [LP:1895067](https://bugs.launchpad.net/curtin/+bug/1895067). If you are able can you try the related branch?

Reply:
Thanks, will look through this today, and report back here, and with customer